pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -668,15 +668,6 @@ func keyfileWatcher(ctx context.Context, watcher *fswatcher.Watcher, keyfilePath
 			// package
 			node.SetIPsecKeyIdentity(spi)
 
-			// NodeValidateImplementation will eventually call
```

Comment:
These calls were deleted as `SetIPsecKeyIdentity` now notifies both nodediscovery (to update the CiliumNode), and datapath.